src/drivers/bitbucket_cloud.js

```diff
@@ -170,6 +197,22 @@ class BitBucketCloud {
     return await response.json();
   }
 
+  async paginatedRequest(opts = {}) {
+    const { method = 'GET', body } = opts;
+    const result = await this.request(opts);
```

Comment:
desstructure

```js
const { next: fullEndpoint } =  await this.request(opts);
```

Reply:
Which one is good? up or down?

Reply:
If you mean my double (uo|down)vote, both are intended. 😄  I wanted to say that I liked the idea of destructuring there, but that the suggested snippet was wrong. Nevermind.

Reply:
Why was wrong ? I did not follow the code but I saw this

```js
fullEndpoint: result.next,
```

and I inferred this

```js
const { next: fullEndpoint } =  await this.request(opts);
```

where you declare next as fullEndpoint

Reply:
It's a tricky one: we also use `result.values` at the end, not only `result.next` as you inferred.

```javascript
const { next, values } =  await this.request(opts);
```

> _This said again without fully reading the code :)_

If you achieve (nearly) linter-level precision without reading the code, I can't imagine what you would do when fully reading the code. 😂